.travis.yml

```diff
@@ -5,5 +5,6 @@ install:
   - npm install
   - npm install -g codecov
 script:
+  - npm flow
```

Comment:
`npm flow` isn't going to work, `npm run flow` will, though!

Reply:
go catch